docker-compose.yml

```diff
@@ -28,14 +28,40 @@ services:
       - '5672:5672'
       - '15672:15672'
   mongo:
-    image: mongo:4.2.9
+    image: mongo:6.0.12
     container_name: metis-sandbox-mongo
     environment:
       MONGO_INITDB_DATABASE: metis-sandbox
       MONGO_INITDB_ROOT_USERNAME: guest
       MONGO_INITDB_ROOT_PASSWORD: guest
     ports:
       - '27017:27017'
+  solr:
+    build:
+      context: docker/solr/
+      dockerfile: Dockerfile
+    container_name: metis-sandbox-solr
+    ports:
+      - "8983:8983"
+      - "9983:9983"
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr
+      - start
+      - -c
+      - -f
+
+  localstack-s3:
+    image: localstack/localstack:s3-latest
+    environment:
+      - DEBUG=${DEBUG:-0}
```

Comment:
?

Reply:
got it from the documentation. https://docs.localstack.cloud/user-guide/aws/s3/
![afbeelding](https://github.com/user-attachments/assets/dd89e2bc-ceec-4a4f-b310-19af62c60920)

Reply:
Perhaps document or plainly remove. This can be added on a per case basis.

Reply:
removed. done.